### DISCUSSION TITLE
BierBot Bricks on RaspberryPi questions

### DISCUSSION THREAD

**Discussion by JaapdBoer** [BernhardSchlegel/BierBot-Bricks#21] [Q&A] [answered]
Hello everyone, 

Hopefully someone can help me with these questions.
My current setup for regulating the fermentation is through a brewpiremix and it is not behaving like I want to. 
So I found this software , it looks really cool and easy and I want to try it on my raspberry pi. (I would also like to build a brick for mashing but that will be a future project)

Now my questions are:
1. Currently I am using this setup
![BrewPi-Remix-v2](https://user-images.githubusercontent.com/91823124/135747527-c9bf3568-5135-45ec-a79c-97fd0b3c54fc.png)

and I was wondering if it would be as easy to keep everything as is, remove the arduino and rewire it like it is shown in the blog? Only using the relais, one-wires, outlet and raspberry pi.
![wiring_rpi_v2_1920_1080-1024x584](https://user-images.githubusercontent.com/91823124/135747581-aa4a6fb6-dad8-4220-9897-03af218507c2.png)

2. It was also mentioned that a display is not necessary but how will I be able to connect to the pi. Would it be possible through a local webhost, like brewpi?

3. Is it possible to add bricks/ extra equipment just like in the webbrowser software or does it need to be wired to the pi? I think the latter bis true but a confirmation would be nice and would it be easy to do so?

**Comment by BernhardSchlegel** [comment, marked as answer]
Hey there,

thanks for considering the BierBot Bricks software :) Here are your answers

1. Yep, that would work. Only issue: The "free" version currently only supports 3 devices and 3 interfaces for each device. So 2x temp + 2x relais on the Rasperry Pi will not work. However, you can setup a Brick and use that temperature sensor (drag and drop from RasberryPi / Brick to any target device works).
2. The display is connected either via the Display header or Raspberry Pi. The BierBot Bricks RaspberryPi client can be configured to open the BierBot Bricks status Website for a nice overview over the brewing process.
3. Nope, you can mix your RaspberryPi with two other Bricks in the free version. You can then freely assign each relay / temperature sensor "in the pool" to any device (masthun, BIAB, fermenter, soon HERMS).

Hope that helps!

Now question from me: Which tool did you use to draw your system from your point #1?

Cheers,
Bernhard.

**Reply by JaapdBoer** [reply]
Hey Bernhard,

Thanks for the fast reply, even on a sunday!

The image isn't mine. I should have also mentioned the source in the initial post to clarify.
It is from https://www.brewpiremix.com/brewpi-hardware/ 

Sorry I can't help you with you question.

**Comment by BernhardSchlegel** [comment]
Ok, now worries. Hope your questions are answered ok.

Enjoy your Sunday !

**Reply by JaapdBoer** [reply]
There are so thank you very much and have a nice Sunday as well!
